README.md

```diff
@@ -81,7 +92,7 @@ As of React Native 0.75.x, the template that used to reside in the `react-native
 
 > _Note:_ The new TV template will only build apps for Apple TV and Android TV. Multiple platform targets are no longer supported in React Native app Podfiles.
 
-To create a new project:
+To create a new project for RNTV 0.76:
```

Comment:
Until RNTV 0.76 is released, you can test new project creation with the community CLI by doing

```
npx @react-native-community/cli@latest init TVTest --template @react-native-tvos/template-tv@next
```